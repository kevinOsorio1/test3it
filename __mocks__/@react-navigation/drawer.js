module.exports = {
  createDrawerNavigator() {
    return {
      Navigator(props) {
        return props.children;
      },
      Screen() {
        return null;
      },
    };
  },
};
