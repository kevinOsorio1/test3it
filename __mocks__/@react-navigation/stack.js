module.exports = {
  createNativeStackNavigator() {
    return {
      Navigator(props) {
        return props.children;
      },
      Screen() {
        return null;
      },
    };
  },
};
