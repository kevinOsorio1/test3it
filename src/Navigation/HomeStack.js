import React from 'react';
import {createNativeStackNavigator} from '@react-navigation/native-stack';
import Home from '../Views/Home/Home';
import IndicatorDetailsView from '../Views/IndicatorDetailsView/IndicatorDetailsView';
import IndicatorTimeLine from '../Views/IndicatorTimeLine/IndicatorTimeLine';

const {Screen, Navigator} = createNativeStackNavigator();

const HomeStack = () => {
  return (
    <Navigator>
      <Screen
        options={{headerShown: false}}
        name="Indicadores"
        component={Home}
      />
      <Screen
        options={{headerShown: false}}
        name="Detalle"
        component={IndicatorDetailsView}
      />
      <Screen
        options={{headerShown: false}}
        name="TimeLine"
        component={IndicatorTimeLine}
      />
    </Navigator>
  );
};

export default HomeStack;
