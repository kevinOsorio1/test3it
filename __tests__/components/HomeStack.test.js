import {NavigationContainer} from '@react-navigation/native';
import {render} from '@testing-library/react-native';
import React from 'react';
import HomeStack from '../../src/Navigation/HomeStack';

test('should render stack', () => {
  const {toJSON} = render(
    <NavigationContainer>
      <HomeStack />
    </NavigationContainer>,
  );
  expect(toJSON).toMatchSnapshot();
});
