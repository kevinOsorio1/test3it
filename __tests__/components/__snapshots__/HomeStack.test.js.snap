// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render stack 1`] = `[Function]`;
